index.bs

```diff
@@ -195,6 +195,19 @@ the user-agent should retain the previously computed layout state if possible.
   values are added on top of any existing containment values.
 </div>
 
+Restrictions and Clarifications {#restrictions}
+===============================
+
+* In situations where [=layout containment=] has no effect (e.g. the
+    element does not generate a principal box), 'subtree-visibility' values also
+    have no effect. In other words, if [=layout containment=] has no effect, the
```

Comment:
Consider removing the second sentence, seems redundant.

Reply:
Done.